dali/dafilesrv/dafilesrv.cpp

```diff
@@ -349,11 +352,14 @@ int main(int argc,char **argv)
     StringBuffer logDir;
     StringBuffer instanceName;
 
-   //Get SSL Settings
+    // Get SSL Settings
+    SSLCfg          useSSL;
+    unsigned short  port;
+    unsigned short  sslport;
     const char *    sslCertFile;
-    bool            useSSL;
-    unsigned short  dafsPort;//DAFILESRV_PORT or SECURE_DAFILESRV_PORT
-    querySecuritySettings(&useSSL, &dafsPort, &sslCertFile, NULL);
+    const char *    sslKeyFile;
+    const char *    sslPassPhrase;
+    queryDafsServerSecSettings(&useSSL, &port, &sslport, &sslCertFile, &sslKeyFile, &sslPassPhrase);
```

Comment:
afaics most of these aren't used.. so should pass nullptr for ones that aren't needed?

It's bit odd, as these aren't used, but are refetched and used in sockfile.cpp by createSecureSocket.